.github/workflows/lint.yml

```
@@ -8,7 +8,7 @@ on:
 
 jobs:
   standardrb:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

I suggest pinning to an exact version for CI reproducibility (the same way we pin gems with Gemfile.lock); a new release of Ubuntu might break the workflow.

```suggestion
    runs-on: ubuntu-24.04
```